css/login.css

```diff
@@ -0,0 +1,83 @@
+.container-main {
+  height: calc(100vh - 6rem);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.form-wrapper {
+  border: 0.1rem solid var(--dark);
+  height: 50rem;
+  width: 40rem;
+  padding: 2rem;
+}
+
+.form-heading {
+  display: flex;
+  align-items: center;
+  display: flex;
```

Comment:
added display flex twice

Reply:
ohh, my bad! 👍🏼 . i'll fix that for sure.